packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Fixed
+  * Fixed scroll issues releated to pinned columns and fixed column header row.
```

Comment:
Are these 2 entries? then these should be on separate lines.

Reply:
It is all one change.  I just called out both issues explicitly.  Is that acceptable?